PP_APP/server/frontend/templates/home.html

```diff
@@ -7,27 +7,30 @@
     <div class="container d-flex h-100 align-items-center">
         <div class="col-sm text-center justify-content-center">
               <h1 class="head-shadow mx-auto mt-2 mb-5">
-                  Eliminate pointless packaging <br>using computer vision
+                  Eliminate pointless packaging <br>using Computer Vision
```

Comment:
Computer vision is not a pronoun so it is not capitalized. I would suggest capatlziign the whole header or removing the capitals on computer vision 

https://en.wikipedia.org/wiki/Computer_vision